versionedRouter.js

```diff
@@ -167,6 +180,9 @@ if (startDestTransformer) {
                 error: error.message,
                 metadata: commonMetadata
               });
+              stats.counter("user_transform_errors", destEvents.length, {
```

Comment:
`user_transform_requests` to count number of requests
`user_transform_failed_requests` to count the number of failed requests.

Similarly
`dest_transform_requests`
`dest_transform_failed_requests`

Reply:
`user_transform_input_events` to count number of events received
`user_transform_output_events` to count number of events sent back

`dest_transform_input_events` to count number of events received
`dest_transform_output_events` to count number of events sent back